libraries/freertos_plus/standard/freertos_plus_tcp/source/portable/NetworkInterface/Zynq/x_emacpsif_dma.c

```diff
@@ -56,7 +56,7 @@ CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
 #endif
 #define TX_OFFSET				ipconfigPACKET_FILLER_SIZE
 
-#define RX_BUFFER_ALIGNMENT	14
+#define dmaRX_TX_BUFFER_SIZE			1536
```

Comment:
Should this instead be a common #define for Ethernet?

Reply:
+1